evap/static/css/evap.scss

```diff
@@ -426,6 +430,15 @@ td.vote-question.choice-error {
     white-space: normal;
     line-height: 1.1;
     width: 90px;
+    border-color: $medium-gray;
+    background-color: $lighter-gray;
+}
+.vote-btn.active {
+    color: $black;
+    box-shadow: inset 0 0 5px $dark-gray;
+}
+.vote-btn:hover {
+    background-color: lighten($light-gray, 5%);
```

Comment:
what do these changes do?

Reply:
active `btn-light` buttons looks differently after the update, this recreates the previous design for the  vote buttons.